implementation/src/main/java/io/smallrye/health/SmallRyeHealthReporter.java

```diff
@@ -69,9 +99,40 @@
     @ConfigProperty(name = "io.smallrye.health.emptyChecksOutcome", defaultValue = "UP")
     String emptyChecksOutcome;
 
-    private List<HealthCheck> additionalChecks = new ArrayList<>();
+    @Inject
+    @ConfigProperty(name = "io.smallrye.health.timeout.seconds", defaultValue = "300")
```

Comment:
300 seconds? That's 5 minutes... isn't this a bit too high?

Reply:
Shouldn't this property be documented somewhere? (I don't know how you manage doc).

Reply:
We don't have documentation yet but all properties will be there. What would be the best default for timeout? I'll change it to one minute for now.